documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -325,3 +325,49 @@ withCredentials([string(credentialsId: 'TOKEN_ID', variable: 'secret')]) {
   """
 }
 ```
+
+## Established Jenkins pipelines
+
+### PR
+
+#### Herbert
+
```

Comment:
For this Herbert PR diagram, a suitable legend could be (in the text) "This diagram shows the process of triggering and running a build of Herbert based on the push of a developer's branch to origin". 

I'm not quite clear on the term "last push". That might mean "previous push", which I don't think you mean. "Latest push" might be better. I don't know if it is worth documenting here what would happen if two pushes were made at different times.

As the push creates a PR, and as the PR tags are associated with the PR, it may be more sensible to have the PR tags associated with the PR rather than the GitHub action, and for the la(te)st push also to feed into the PR rather than the GitHub action. Alternatively, if the PR tags are associated with the PR build rather than the PR, an extended label like "PR pipeline tags" feeding into the action would do the job.

I think Nick's point that the four build boxes are not exhaustive and maybe should be more generic is a good one.

To go with my comments on the Horace diagram below, it might be worth making the Build box say "Herbert Build" to be explicit about the process going on.